gems/aws-sdk-core/lib/aws-sdk-core/plugins/user_agent.rb

```diff
@@ -23,12 +39,15 @@ class UserAgent < Seahorse::Client::Plugin
         app_id
       end
 
-      def self.feature(feature, &block)
-        Thread.current[:aws_sdk_core_user_agent_feature] ||= []
-        Thread.current[:aws_sdk_core_user_agent_feature] << "ft/#{feature}"
+      def self.metric(metric, &block)
+        Thread.current[:aws_sdk_core_user_agent_metric] ||= []
```

Comment:
Is there an old service gem (relying on `:aws_sdk_core_user_agent_feature`) + new core risk here?

Reply:
Oh.. yeah I think so. I think feature() will have to persist but as a no-op.